CHANGELOG.md

```diff
@@ -1,3 +1,17 @@
+### 1.7.0
+
+- Adds a new method, `canMakePayments`, that provides a way to check if the current user is allowed to make purchases on the device. 
+    https://github.com/RevenueCat/purchases-hybrid-common/pull/77
```

Comment:
@beylmk just double-checking that this is ready to go

Reply:
@aboedo yep! i was gonna release today actually haha but wanted to ask you about the 1.6.3 release. thanks for taking care of this!